lib/fog/google/models/pubsub/subscription.rb

```diff
@@ -0,0 +1,90 @@
+require "base64"
+require "fog/core/model"
+require "fog/google/models/pubsub/received_message"
```

Comment:
Yeah. The thing about it is that we don't need it anymore thus it is kind of bogus and a legacy of when this was still inside the main `fog` gem.

So mostly of the approaches and stuff we are doing is move to a solution that was needed inside the main `fog` gem to a solution more accurate to a standalone gem. e.g `autoload`.